src/d3a/models/state.py

```diff
@@ -237,6 +336,19 @@ class PVState(ProductionState):
     Completely inherits ProductionState, but we keep this class for backward compatibility.
     """
 
+    def _calculate_unsettled_energy_kWh(
+            self, measured_energy_kWh: float, time_slot: DateTime) -> float:
+        """
+        Returns negative values for overproduction (offer will be placed on the settlement market)
+        and positive values for underproduction (bid will be placed on the settlement market)
+        :param measured_energy_kWh: Measured energy that the PV produced
+        :param time_slot: time slot of the measured energy
+        :return: Deviation between forecasted and measured energy
```

Comment:
```suggestion
        :return: Deviation between energy traded in spot market traded and measured energy
```

Reply:
@spyrostz please add this in the frame of 3568. Thanks!